server/aesdsocket.c

```diff
@@ -91,62 +92,62 @@ int main(int argc, char **argv)
     int ret_status = 0;
     int deamon_enabled = 0;
 
-    openlog(NULL, LOG_CONS | LOG_PID | LOG_PERROR, LOG_USER);									/*open connection for sys logging, ident is NULL to use this Program for the user level messages*/
-    
-    if((signal(SIGINT, sig_handler) == SIG_ERR) || (signal(SIGTERM, sig_handler) == SIG_ERR))
+    openlog(NULL, LOG_CONS | LOG_PID | LOG_PERROR, LOG_USER); /*open connection for sys logging, ident is NULL to use this Program for the user level messages*/
+
+    if ((signal(SIGINT, sig_handler) == SIG_ERR) || (signal(SIGTERM, sig_handler) == SIG_ERR))
     {
-        syslog(LOG_ERR," Signal handler error");
+        syslog(LOG_ERR, " Signal handler error");
         cleanup_on_exit();
         exit(EXIT_FAILURE);
     }
 
     /* Check for daemon enabled*/
-    if(argc == 2)
+    if (argc == 2)
     {
-        if(strcmp(argv[1],"-d") == 0)
+        if (strcmp(argv[1], "-d") == 0)
         {
             deamon_enabled = 1;
-            syslog(LOG_INFO,"Running this process as Daemon");
+            syslog(LOG_INFO, "Running this process as Daemon");
         }
         else
         {
-            syslog(LOG_ERR,"Invalid 2nd argument for aesdsocket");
+            syslog(LOG_ERR, "Invalid 2nd argument for aesdsocket");
             return (EXIT_FAILURE);
         }
     }
-    else if(argc > 2)
+    else if (argc > 2)
     {
-        syslog(LOG_ERR,"Invalid Number of arguments for aesdsocket");
+        syslog(LOG_ERR, "Invalid Number of arguments for aesdsocket");
         return (EXIT_FAILURE);
     }
 
     /*if the run as daemon argument is present run daemon*/
-    if(deamon_enabled == 1)
+    if (deamon_enabled == 1)
     {
-        if(daemon(0,0) == -1)
+        if (daemon(0, 0) == -1)
         {
-            syslog(LOG_ERR,"Daemon fork failure");
+            syslog(LOG_ERR, "Daemon fork failure");
             cleanup_on_exit();
             return (EXIT_FAILURE);
         }
     }
 
     /* open file for Socket data*/
     fd_wr_file = open(SOCK_FILE_WRITE, O_CREAT | O_RDWR | O_APPEND, S_IRWXU | S_IRWXG | S_IRWXO);
-    if(fd_wr_file == -1)
+    if (fd_wr_file == -1)
```

Comment:
This would also explain why the content from the sockettest run fails, since the driver circular buffer will continue to have the content from the drivertest step.